src/routes/RequestsRoute.js

```diff
@@ -1211,11 +1256,9 @@ class RequestsRoute extends React.Component {
     })
   );
 
-  onBeforePrintForPrintButton = (slipType, slipsData) => {
-    if (slipType === SLIPS_TYPE.PICK_SLIP) { // TODO: add enable print log settings check
-      const requestIds = extractPickSlipRequestIds(slipsData);
-      this.savePrintEventDetails(requestIds);
-    }
+  onBeforePrintForPrintButton = (slipsData) => {
+    const requestIds = extractPickSlipRequestIds(slipsData);
+    this.savePrintEventDetails(requestIds);
```

Comment:
isn't this check no more needed?
// TODO: add enable print log settings check

Reply:
yes not needed now, as it is covered in the function call below.